.github/workflows/slowtest.yaml

```diff
@@ -0,0 +1,63 @@
+name: SlowEst
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+    tags:
+      # YYYYMMDD
+      - "20[0-9][0-9][0-1][0-9][0-3][0-9]*"
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
We probably shouldn't cancel the job on `main` so we would know which commit broke it: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-concurrency#example-only-cancel-in-progress-jobs-on-specific-branches

I also think we can run it on PRs just not make it required for merging. I think if you add a new workflow it will be not required by default.

Reply:
![image](https://github.com/user-attachments/assets/547afe83-8ffe-433e-9877-26f371d00d2c)

indeed doesn't seem to be required

Reply:
39d2e888b70e375be9727249f2dc46ea70617ae9 and 5a609dfc9d20643873302fe82f09a443ff710ac8